cmd/generate.go

```diff
@@ -34,6 +35,10 @@ var generateCmd = &cobra.Command{
 	Use:   "generate",
 	Short: "Generate Kubernetes resources from App definition",
 	Run: func(cmd *cobra.Command, args []string) {
+		if err := validateFiles(AppFiles); err != nil {
+			fmt.Println(errors.Wrap(err, "Error: Unable to validate input files"))
```

Comment:
as pointed by @kadel we can make this error message more specific also see the comment in https://github.com/kedgeproject/kedge/pull/151/files#r127202325